tests/integration/test_host.py

```diff
@@ -44,95 +48,113 @@ async def hostprovider(adcm_client: ADCMClient, hostprovider_bundle: Bundle) ->
+    await cluster.hosts.remove(host)
+    await host.delete()
+
+
+async def test_host_in_host_group(adcm_client: ADCMClient, hostprovider: HostProvider, cluster: Cluster) -> None:
+    for i in range(80):
+        await adcm_client.hosts.create(name=f"test-host-{i}", hostprovider=hostprovider, cluster=cluster)
+
+    await cluster.config_host_groups.create(name="config-host-group", description="config host group description")
+    await cluster.action_host_groups.create(name="action-host-group", description="action host group description")
+
+    config_host_group = await cluster.config_host_groups.get(name__eq="config-host-group")
+    action_host_group = await cluster.action_host_groups.get(name__eq="action-host-group")
+    await config_host_group.hosts.add(await cluster.hosts.filter(name__contains="test-host"))
+    await action_host_group.hosts.add(await cluster.hosts.filter(name__contains="test-host"))
+
+    await _test_host_properties(config_host_group.hosts, hostprovider, cluster)
+    await _test_host_properties(config_host_group.hosts, hostprovider, cluster)
 
+    assert _test_pagination(config_host_group.hosts)
+    assert _test_pagination(action_host_group.hosts)
 
-async def _test_host_properties(adcm_client: ADCMClient, hostprovider: HostProvider, cluster: Cluster) -> None:
-    await adcm_client.hosts.create(name="test-host", description="host description", hostprovider=hostprovider)
-    await cluster.hosts.add(host=await adcm_client.hosts.get(name__eq="test-host"))
+    assert _test_pagination(config_host_group.hosts)
+    assert _test_pagination(action_host_group.hosts)
 
-    host = await adcm_client.hosts.get()
-    assert host.name == "test-host"
+
+async def _test_host_properties(
+    hosts_node: HostsInActionHostGroupNode | HostsInConfigHostGroupNode | HostsNode,
+    hostprovider: HostProvider,
+    cluster: Cluster,
+) -> None:
+    host = await hosts_node.get(name__eq="test-host-0")
+    assert host.name == "test-host-0"
     assert (await host.hostprovider).name == hostprovider.name
     assert (await host.cluster).name == cluster.name  # pyright: ignore[reportOptionalMemberAccess]
     assert isinstance(host.actions, ActionsAccessor)
     assert await host.get_status() == "down"
     assert (await host.maintenance_mode).value == "off"
 
 
-async def _test_host_accessors(adcm_client: ADCMClient, hostprovider: HostProvider, cluster: Cluster) -> None:
-    for new_host in ["host-1", "host-2", "host-3"]:
-        await adcm_client.hosts.create(name=new_host, description="host description", hostprovider=hostprovider)
-
-    host = await adcm_client.hosts.get(name__eq="host-1")
+async def _test_host_accessors(
+    hosts_node: HostsInActionHostGroupNode | HostsInConfigHostGroupNode | HostsNode, cluster: Cluster
+) -> None:
+    host = await hosts_node.get(name__eq="test-host-1")
     assert isinstance(host, Host)
-    assert host.name == "host-1"
+    assert host.name == "test-host-1"
```

Comment:
Why do we need to check the property in the accessors test?

Reply:
removed